gengokit/gengokit.go

```diff
@@ -36,6 +36,8 @@ func init() {
 type templateExecutor struct {
 	// import path for the directory containing the definition .proto files
 	ImportPath string
+	// import path for .pb.go files containing service structs
```

Comment:
Use traditional sentence rules when writing comments that will be exported as documentation. I.e., capitalization, punctuation.